### ISSUE TITLE
GitLab Listener crashes with NPE

### ISSUE THREAD

**Issue by thomseno** [allenta/gitlab-listener#35]
Since recently the following exception is logged when a push event is processed by the Gitlab Listener:

```
2016-05-06 08:42:20,379 http-nio-8180-exec-18 ERROR      [o.a.c.c.C.[.[localhost].[/jira].[servlet-module-container-servlet]] Servlet.service() for servlet [servlet-module-container-servlet] in context with path [/jira] threw exception
java.lang.NullPointerException
    at java.io.StringReader.<init>(StringReader.java:50)
    at java.util.Scanner.<init>(Scanner.java:702)
    at com.allenta.jira.plugins.gitlab.listener.settings.Mapping.parse(Mapping.java:35)
    at com.allenta.jira.plugins.gitlab.listener.Servlet.doPost(Servlet.java:150)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:644)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
    at com.atlassian.plugin.servlet.DelegatingPluginServlet.service(DelegatingPluginServlet.java:37)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
    at com.atlassian.plugin.servlet.ServletModuleContainerServlet.service(ServletModuleContainerServlet.java:45)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
    ... 105 filtered
    at com.atlassian.jira.security.JiraSecurityFilter.doFilter(JiraSecurityFilter.java:70)
    ... 90 filtered
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
    at java.lang.Thread.run(Thread.java:745)
```

**Comment by carlosabalde** [comment]
Thanks for the bug report. That's related with a missing check in the latest feature added to the add-on. A new version fixing this will be released in ~ 15 minutes.

**Comment by carlosabalde** [comment]
Bug should now be fixed in GitLab Listener [1.2.8](https://marketplace.atlassian.com/plugins/com.allenta.jira.plugins.gitlab.gitlab-listener/versions#b100200800) (JIRA 6.x) and [2.0.8](https://marketplace.atlassian.com/plugins/com.allenta.jira.plugins.gitlab.gitlab-listener/versions#b200000800) (JIRA 7.x). Thanks again for reporting!

**Comment by thomseno** [comment]
Thanks for fixing so fast 👍🏻
